src/iuse.cpp

```diff
@@ -4954,24 +4954,6 @@ int iuse::arrow_flamable(player *p, item *it, bool, const tripoint& )
     return 1;
 }
 
-int iuse::molotov(player *p, item *it, bool, const tripoint& )
-{
-    if (p->is_underwater()) {
```

Comment:
That was something I was worried about, but currently nothing using iuse::transform is capable of being used underwater; doesn't matter if it uses fire or not. I was debating opening an issue on that, but we really don't have enough underwater content.